Binary files /tmp/cracklib/.cracklib.SlackBuild.un~ and /source/a/cracklib/.cracklib.SlackBuild.un~ differ
diff -ruBN /tmp/cracklib/cracklib.SlackBuild /source/a/cracklib/cracklib.SlackBuild
--- /tmp/cracklib/cracklib.SlackBuild	2021-01-03 16:52:55.000000000 +0000
+++ /source/a/cracklib/cracklib.SlackBuild	2021-01-03 16:53:11.000000000 +0000
@@ -58,7 +58,7 @@
   LIBDIRSUFFIX=""
 elif [ "$ARCH" = "x86_64" ]; then
   SLKCFLAGS="-O2 -fPIC"
-  LIBDIRSUFFIX="64"
+  LIBDIRSUFFIX=""
 else
   SLKCFLAGS="-O2"
   LIBDIRSUFFIX=""
@@ -90,7 +90,7 @@
   --disable-static \
   --without-python \
   --disable-nls \
-  --build=$ARCH-slackware-linux || exit 1
+  --build=$ARCH-slackware-linux-musl || exit 1
 
 make $NUMJOBS || exit 1
 make install DESTDIR=$PKG || exit 1
diff -ruBN /tmp/cracklib/cracklib.SlackBuild.log /source/a/cracklib/cracklib.SlackBuild.log
--- /tmp/cracklib/cracklib.SlackBuild.log	1970-01-01 00:00:00.000000000 +0000
+++ /source/a/cracklib/cracklib.SlackBuild.log	2021-01-03 16:53:18.000000000 +0000
@@ -0,0 +1,575 @@
+cracklib-2.9.7/
+cracklib-2.9.7/config.rpath
+cracklib-2.9.7/config.h.in
+cracklib-2.9.7/install-sh
+cracklib-2.9.7/Makefile.in
+cracklib-2.9.7/ABOUT-NLS
+cracklib-2.9.7/util/
+cracklib-2.9.7/util/testlib.c
+cracklib-2.9.7/util/Makefile.in
+cracklib-2.9.7/util/packer.c
+cracklib-2.9.7/util/teststr.c
+cracklib-2.9.7/util/testnum.c
+cracklib-2.9.7/util/Makefile.am
+cracklib-2.9.7/util/check.c
+cracklib-2.9.7/util/unpacker.c
+cracklib-2.9.7/util/cracklib-format
+cracklib-2.9.7/util/create-cracklib-dict
+cracklib-2.9.7/README
+cracklib-2.9.7/po/
+cracklib-2.9.7/po/be.po
+cracklib-2.9.7/po/or.gmo
+cracklib-2.9.7/po/zh_CN.po
+cracklib-2.9.7/po/insert-header.sin
+cracklib-2.9.7/po/hi.po
+cracklib-2.9.7/po/Rules-quot
+cracklib-2.9.7/po/lt.gmo
+cracklib-2.9.7/po/id.po
+cracklib-2.9.7/po/de.gmo
+cracklib-2.9.7/po/ru.gmo
+cracklib-2.9.7/po/nb.po
+cracklib-2.9.7/po/boldquot.sed
+cracklib-2.9.7/po/remove-potcdate.sin
+cracklib-2.9.7/po/pl.po
+cracklib-2.9.7/po/ca.gmo
+cracklib-2.9.7/po/id.gmo
+cracklib-2.9.7/po/kk.gmo
+cracklib-2.9.7/po/ko.gmo
+cracklib-2.9.7/po/sq.po
+cracklib-2.9.7/po/pt_BR.gmo
+cracklib-2.9.7/po/te.po
+cracklib-2.9.7/po/te.gmo
+cracklib-2.9.7/po/as.gmo
+cracklib-2.9.7/po/kk.po
+cracklib-2.9.7/po/POTFILES.in
+cracklib-2.9.7/po/eu.gmo
+cracklib-2.9.7/po/zh_TW.po
+cracklib-2.9.7/po/es.gmo
+cracklib-2.9.7/po/ru.po
+cracklib-2.9.7/po/ca.po
+cracklib-2.9.7/po/nl.gmo
+cracklib-2.9.7/po/or.po
+cracklib-2.9.7/po/fi.po
+cracklib-2.9.7/po/fi.gmo
+cracklib-2.9.7/po/uk.po
+cracklib-2.9.7/po/zh_CN.gmo
+cracklib-2.9.7/po/pt.po
+cracklib-2.9.7/po/tr.po
+cracklib-2.9.7/po/el.po
+cracklib-2.9.7/po/sq.gmo
+cracklib-2.9.7/po/de.po
+cracklib-2.9.7/po/pt_BR.po
+cracklib-2.9.7/po/mr.po
+cracklib-2.9.7/po/he.gmo
+cracklib-2.9.7/po/ko.po
+cracklib-2.9.7/po/mr.gmo
+cracklib-2.9.7/po/cracklib.pot
+cracklib-2.9.7/po/hi.gmo
+cracklib-2.9.7/po/ja.gmo
+cracklib-2.9.7/po/bn_IN.po
+cracklib-2.9.7/po/quot.sed
+cracklib-2.9.7/po/it.gmo
+cracklib-2.9.7/po/zh_TW.gmo
+cracklib-2.9.7/po/hu.gmo
+cracklib-2.9.7/po/Makevars
+cracklib-2.9.7/po/en@boldquot.header
+cracklib-2.9.7/po/gu.po
+cracklib-2.9.7/po/tr.gmo
+cracklib-2.9.7/po/sv.gmo
+cracklib-2.9.7/po/pa.po
+cracklib-2.9.7/po/pa.gmo
+cracklib-2.9.7/po/sk.po
+cracklib-2.9.7/po/as.po
+cracklib-2.9.7/po/pl.gmo
+cracklib-2.9.7/po/uk.gmo
+cracklib-2.9.7/po/da.gmo
+cracklib-2.9.7/po/kn.gmo
+cracklib-2.9.7/po/cs.po
+cracklib-2.9.7/po/hu.po
+cracklib-2.9.7/po/nb.gmo
+cracklib-2.9.7/po/kn.po
+cracklib-2.9.7/po/pt.gmo
+cracklib-2.9.7/po/eu.po
+cracklib-2.9.7/po/ml.gmo
+cracklib-2.9.7/po/cs.gmo
+cracklib-2.9.7/po/ja.po
+cracklib-2.9.7/po/fr.gmo
+cracklib-2.9.7/po/bn_IN.gmo
+cracklib-2.9.7/po/it.po
+cracklib-2.9.7/po/sv.po
+cracklib-2.9.7/po/sl.gmo
+cracklib-2.9.7/po/ml.po
+cracklib-2.9.7/po/be.gmo
+cracklib-2.9.7/po/ta.gmo
+cracklib-2.9.7/po/en@quot.header
+cracklib-2.9.7/po/he.po
+cracklib-2.9.7/po/sk.gmo
+cracklib-2.9.7/po/stamp-po
+cracklib-2.9.7/po/sl.po
+cracklib-2.9.7/po/nl.po
+cracklib-2.9.7/po/es.po
+cracklib-2.9.7/po/lt.po
+cracklib-2.9.7/po/el.gmo
+cracklib-2.9.7/po/gu.gmo
+cracklib-2.9.7/po/ta.po
+cracklib-2.9.7/po/Makefile.in.in
+cracklib-2.9.7/po/fr.po
+cracklib-2.9.7/po/da.po
+cracklib-2.9.7/python/
+cracklib-2.9.7/python/Makefile.in
+cracklib-2.9.7/python/setup.py.in
+cracklib-2.9.7/python/Makefile.am
+cracklib-2.9.7/python/test_cracklib.py
+cracklib-2.9.7/python/cracklib.py
+cracklib-2.9.7/python/_cracklib.c
+cracklib-2.9.7/configure.ac
+cracklib-2.9.7/README-DAWG
+cracklib-2.9.7/m4/
+cracklib-2.9.7/m4/nls.m4
+cracklib-2.9.7/m4/Makefile.in
+cracklib-2.9.7/m4/intl.m4
+cracklib-2.9.7/m4/printf-posix.m4
+cracklib-2.9.7/m4/po.m4
+cracklib-2.9.7/m4/inttypes-pri.m4
+cracklib-2.9.7/m4/libtool.m4
+cracklib-2.9.7/m4/wint_t.m4
+cracklib-2.9.7/m4/intdiv0.m4
+cracklib-2.9.7/m4/lock.m4
+cracklib-2.9.7/m4/lt~obsolete.m4
+cracklib-2.9.7/m4/lib-ld.m4
+cracklib-2.9.7/m4/inttypes_h.m4
+cracklib-2.9.7/m4/intlmacosx.m4
+cracklib-2.9.7/m4/lib-link.m4
+cracklib-2.9.7/m4/ltsugar.m4
+cracklib-2.9.7/m4/codeset.m4
+cracklib-2.9.7/m4/glibc21.m4
+cracklib-2.9.7/m4/size_max.m4
+cracklib-2.9.7/m4/intmax.m4
+cracklib-2.9.7/m4/ltoptions.m4
+cracklib-2.9.7/m4/Makefile.am
+cracklib-2.9.7/m4/iconv.m4
+cracklib-2.9.7/m4/wchar_t.m4
+cracklib-2.9.7/m4/gettext.m4
+cracklib-2.9.7/m4/progtest.m4
+cracklib-2.9.7/m4/xsize.m4
+cracklib-2.9.7/m4/intldir.m4
+cracklib-2.9.7/m4/glibc2.m4
+cracklib-2.9.7/m4/lib-prefix.m4
+cracklib-2.9.7/m4/ltversion.m4
+cracklib-2.9.7/m4/stdint_h.m4
+cracklib-2.9.7/m4/lcmessage.m4
+cracklib-2.9.7/m4/longlong.m4
+cracklib-2.9.7/m4/visibility.m4
+cracklib-2.9.7/m4/uintmax_t.m4
+cracklib-2.9.7/cracklib.spec.in
+cracklib-2.9.7/configure
+cracklib-2.9.7/COPYING.LIB
+cracklib-2.9.7/ChangeLog
+cracklib-2.9.7/config.guess
+cracklib-2.9.7/doc/
+cracklib-2.9.7/doc/Makefile.in
+cracklib-2.9.7/doc/Makefile.am
+cracklib-2.9.7/doc/cracklib.magic
+cracklib-2.9.7/aclocal.m4
+cracklib-2.9.7/test-data
+cracklib-2.9.7/depcomp
+cracklib-2.9.7/ltmain.sh
+cracklib-2.9.7/Makefile.am
+cracklib-2.9.7/lib/
+cracklib-2.9.7/lib/Makefile.in
+cracklib-2.9.7/lib/fascist.c
+cracklib-2.9.7/lib/packlib.c
+cracklib-2.9.7/lib/crack.h
+cracklib-2.9.7/lib/stringlib.c
+cracklib-2.9.7/lib/Makefile.am
+cracklib-2.9.7/lib/packer.h
+cracklib-2.9.7/lib/rules.c
+cracklib-2.9.7/compile
+cracklib-2.9.7/config.sub
+cracklib-2.9.7/py-compile
+cracklib-2.9.7/README-LICENSE
+cracklib-2.9.7/INSTALL
+cracklib-2.9.7/AUTHORS
+cracklib-2.9.7/README-WORDS
+cracklib-2.9.7/dicts/
+cracklib-2.9.7/dicts/Makefile.in
+cracklib-2.9.7/dicts/cracklib-small
+cracklib-2.9.7/dicts/Makefile.am
+cracklib-2.9.7/NEWS
+cracklib-2.9.7/missing
+checking for a BSD-compatible install... /usr/bin/ginstall -c
+checking whether build environment is sane... yes
+checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
+checking for gawk... gawk
+checking whether make sets $(MAKE)... yes
+checking whether make supports nested variables... yes
+checking build system type... x86_64-slackware-linux-musl
+checking host system type... x86_64-slackware-linux-musl
+checking how to print strings... printf
+checking for style of include used by make... GNU
+checking for gcc... gcc
+checking whether the C compiler works... yes
+checking for C compiler default output file name... a.out
+checking for suffix of executables... 
+checking whether we are cross compiling... no
+checking for suffix of object files... o
+checking whether we are using the GNU C compiler... yes
+checking whether gcc accepts -g... yes
+checking for gcc option to accept ISO C89... none needed
+checking whether gcc understands -c and -o together... yes
+checking dependency style of gcc... gcc3
+checking for a sed that does not truncate output... /usr/bin/sed
+checking for grep that handles long lines and -e... /usr/bin/grep
+checking for egrep... /usr/bin/grep -E
+checking for fgrep... /usr/bin/grep -F
+checking for ld used by gcc... /usr/bin/ld
+checking if the linker (/usr/bin/ld) is GNU ld... yes
+checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
+checking the name lister (/usr/bin/nm -B) interface... BSD nm
+checking whether ln -s works... yes
+checking the maximum length of command line arguments... 98304
+checking how to convert x86_64-slackware-linux-musl file names to x86_64-slackware-linux-musl format... func_convert_file_noop
+checking how to convert x86_64-slackware-linux-musl file names to toolchain format... func_convert_file_noop
+checking for /usr/bin/ld option to reload object files... -r
+checking for objdump... objdump
+checking how to recognize dependent libraries... pass_all
+checking for dlltool... dlltool
+checking how to associate runtime and link libraries... printf %s\n
+checking for ar... ar
+checking for archiver @FILE support... @
+checking for strip... strip
+checking for ranlib... ranlib
+checking command to parse /usr/bin/nm -B output from gcc object... ok
+checking for sysroot... no
+checking for a working dd... /usr/bin/dd
+checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
+checking for mt... mt
+checking if mt is a manifest tool... no
+checking how to run the C preprocessor... gcc -E
+checking for ANSI C header files... yes
+checking for sys/types.h... yes
+checking for sys/stat.h... yes
+checking for stdlib.h... yes
+checking for string.h... yes
+checking for memory.h... yes
+checking for strings.h... yes
+checking for inttypes.h... yes
+checking for stdint.h... yes
+checking for unistd.h... yes
+checking for dlfcn.h... yes
+checking for objdir... .libs
+checking if gcc supports -fno-rtti -fno-exceptions... no
+checking for gcc option to produce PIC... -fPIC -DPIC
+checking if gcc PIC flag -fPIC -DPIC works... yes
+checking if gcc static flag -static works... yes
+checking if gcc supports -c -o file.o... yes
+checking if gcc supports -c -o file.o... (cached) yes
+checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
+checking whether -lc should be explicitly linked in... no
+checking dynamic linker characteristics... GNU/Linux ld.so
+checking how to hardcode library paths into programs... immediate
+checking whether stripping libraries is possible... yes
+checking if libtool supports shared libraries... yes
+checking whether to build shared libraries... yes
+checking whether to build static libraries... no
+checking for gcc... (cached) gcc
+checking whether we are using the GNU C compiler... (cached) yes
+checking whether gcc accepts -g... (cached) yes
+checking for gcc option to accept ISO C89... (cached) none needed
+checking whether gcc understands -c and -o together... (cached) yes
+checking dependency style of gcc... (cached) gcc3
+checking how to run the C preprocessor... gcc -E
+checking whether ln -s works... yes
+checking whether make sets $(MAKE)... (cached) yes
+checking for unistd.h... (cached) yes
+checking fcntl.h usability... yes
+checking fcntl.h presence... yes
+checking for fcntl.h... yes
+checking for inttypes.h... (cached) yes
+checking for stdint.h... (cached) yes
+checking pthread.h usability... yes
+checking pthread.h presence... yes
+checking for pthread.h... yes
+checking zlib.h usability... yes
+checking zlib.h presence... yes
+checking for zlib.h... yes
+checking for library containing gzopen... -lz
+checking if LINE_MAX is defined... yes
+checking for an ANSI C-conforming const... yes
+checking for strdup... yes
+checking for getpwuid_r... yes
+checking whether NLS is requested... no
+checking for msgfmt... /usr/bin/msgfmt
+checking for gmsgfmt... /usr/bin/msgfmt
+checking for xgettext... /usr/bin/xgettext
+checking for msgmerge... /usr/bin/msgmerge
+checking for ld used by GCC... /usr/bin/ld -m elf_x86_64
+checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
+checking for shared library run path origin... done
+checking for CFPreferencesCopyAppValue... no
+checking for CFLocaleCopyCurrent... no
+checking whether to use NLS... no
+checking default dict filename... $(pkgdatadir)/pw_dict
+checking that generated files are newer than configure... done
+configure: creating ./config.status
+config.status: creating util/Makefile
+config.status: creating lib/Makefile
+config.status: creating doc/Makefile
+config.status: creating python/Makefile
+config.status: creating Makefile
+config.status: creating python/setup.py
+config.status: creating po/Makefile.in
+config.status: creating m4/Makefile
+config.status: creating dicts/Makefile
+config.status: creating cracklib.spec
+config.status: creating config.h
+config.status: executing depfiles commands
+config.status: executing libtool commands
+config.status: executing po-directories commands
+config.status: creating po/POTFILES
+config.status: creating po/Makefile
+make  all-recursive
+make[1]: Entering directory '/tmp/cracklib-2.9.7'
+Making all in m4
+make[2]: Entering directory '/tmp/cracklib-2.9.7/m4'
+make[2]: Nothing to be done for 'all'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/m4'
+Making all in lib
+make[2]: Entering directory '/tmp/cracklib-2.9.7/lib'
+/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT fascist.lo -MD -MP -MF .deps/fascist.Tpo -c -o fascist.lo fascist.c
+/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT packlib.lo -MD -MP -MF .deps/packlib.Tpo -c -o packlib.lo packlib.c
+/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT rules.lo -MD -MP -MF .deps/rules.Tpo -c -o rules.lo rules.c
+libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I. -I.. -I../lib -DIN_CRACKLIB -DDEFAULT_CRACKLIB_DICT=\"/usr/share/cracklib/pw_dict\" -Wall -O2 -fPIC -MT rules.lo -MD -MP -MF .deps/rules.Tpo -c rules.c  -fPIC -DPIC -o .libs/rules.o
+libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I. -I.. -I../lib -DIN_CRACKLIB -DDEFAULT_CRACKLIB_DICT=\"/usr/share/cracklib/pw_dict\" -Wall -O2 -fPIC -MT fascist.lo -MD -MP -MF .deps/fascist.Tpo -c fascist.c  -fPIC -DPIC -o .libs/fascist.o
+libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I. -I.. -I../lib -DIN_CRACKLIB -DDEFAULT_CRACKLIB_DICT=\"/usr/share/cracklib/pw_dict\" -Wall -O2 -fPIC -MT packlib.lo -MD -MP -MF .deps/packlib.Tpo -c packlib.c  -fPIC -DPIC -o .libs/packlib.o
+rules.c:7:19: warning: ‘vers_id’ defined but not used [-Wunused-const-variable=]
+    7 | static const char vers_id[] = "rules.c : v5.0p3 Alec Muffett 20 May 1993";
+      |                   ^~~~~~~
+packlib.c:21:19: warning: ‘vers_id’ defined but not used [-Wunused-const-variable=]
+   21 | static const char vers_id[] = "packlib.c : v2.3p2 Alec Muffett 18 May 1993";
+      |                   ^~~~~~~
+mv -f .deps/fascist.Tpo .deps/fascist.Plo
+/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT stringlib.lo -MD -MP -MF .deps/stringlib.Tpo -c -o stringlib.lo stringlib.c
+mv -f .deps/packlib.Tpo .deps/packlib.Plo
+mv -f .deps/rules.Tpo .deps/rules.Plo
+libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I. -I.. -I../lib -DIN_CRACKLIB -DDEFAULT_CRACKLIB_DICT=\"/usr/share/cracklib/pw_dict\" -Wall -O2 -fPIC -MT stringlib.lo -MD -MP -MF .deps/stringlib.Tpo -c stringlib.c  -fPIC -DPIC -o .libs/stringlib.o
+stringlib.c:15:19: warning: ‘vers_id’ defined but not used [-Wunused-const-variable=]
+   15 | static const char vers_id[] = "stringlib.c : v2.3p2 Alec Muffett 18 May 1993";
+      |                   ^~~~~~~
+mv -f .deps/stringlib.Tpo .deps/stringlib.Plo
+/bin/sh ../libtool  --tag=CC   --mode=link gcc  -O2 -fPIC -version-info 11:0:9  -o libcrack.la -rpath /usr/lib fascist.lo packlib.lo rules.lo stringlib.lo  -lz 
+libtool: link: gcc -shared  -fPIC -DPIC  .libs/fascist.o .libs/packlib.o .libs/rules.o .libs/stringlib.o   -lz  -O2   -Wl,-soname -Wl,libcrack.so.2 -o .libs/libcrack.so.2.9.0
+libtool: link: (cd ".libs" && rm -f "libcrack.so.2" && ln -s "libcrack.so.2.9.0" "libcrack.so.2")
+libtool: link: (cd ".libs" && rm -f "libcrack.so" && ln -s "libcrack.so.2.9.0" "libcrack.so")
+libtool: link: ( cd ".libs" && rm -f "libcrack.la" && ln -s "../libcrack.la" "libcrack.la" )
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/lib'
+Making all in util
+make[2]: Entering directory '/tmp/cracklib-2.9.7/util'
+gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT packer.o -MD -MP -MF .deps/packer.Tpo -c -o packer.o packer.c
+gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT unpacker.o -MD -MP -MF .deps/unpacker.Tpo -c -o unpacker.o unpacker.c
+gcc -DHAVE_CONFIG_H -I. -I..  -I. -I.. -I../lib '-DDEFAULT_CRACKLIB_DICT="/usr/share/cracklib/pw_dict"' -Wall   -O2 -fPIC -MT check.o -MD -MP -MF .deps/check.Tpo -c -o check.o check.c
+mv -f .deps/unpacker.Tpo .deps/unpacker.Po
+/bin/sh ../libtool  --tag=CC   --mode=link gcc  -O2 -fPIC   -o cracklib-unpacker unpacker.o ../lib/libcrack.la -lz 
+mv -f .deps/check.Tpo .deps/check.Po
+mv -f .deps/packer.Tpo .deps/packer.Po
+/bin/sh ../libtool  --tag=CC   --mode=link gcc  -O2 -fPIC   -o cracklib-check check.o ../lib/libcrack.la -lz 
+/bin/sh ../libtool  --tag=CC   --mode=link gcc  -O2 -fPIC   -o cracklib-packer packer.o ../lib/libcrack.la -lz 
+libtool: link: gcc -O2 -fPIC -o .libs/cracklib-unpacker unpacker.o  ../lib/.libs/libcrack.so -lz
+libtool: link: gcc -O2 -fPIC -o .libs/cracklib-packer packer.o  ../lib/.libs/libcrack.so -lz
+libtool: link: gcc -O2 -fPIC -o .libs/cracklib-check check.o  ../lib/.libs/libcrack.so -lz
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/util'
+Making all in po
+make[2]: Entering directory '/tmp/cracklib-2.9.7/po'
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/po'
+Making all in doc
+make[2]: Entering directory '/tmp/cracklib-2.9.7/doc'
+make[2]: Nothing to be done for 'all'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/doc'
+Making all in python
+make[2]: Entering directory '/tmp/cracklib-2.9.7/python'
+make[2]: Nothing to be done for 'all'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/python'
+Making all in dicts
+make[2]: Entering directory '/tmp/cracklib-2.9.7/dicts'
+make[2]: Nothing to be done for 'all'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/dicts'
+make[2]: Entering directory '/tmp/cracklib-2.9.7'
+
+=======================================================
+Be sure and obtain a large wordlist file and run
+make install and make dict to generate the word
+list index file. See the README file for more details.
+=======================================================
+
+make[2]: Leaving directory '/tmp/cracklib-2.9.7'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7'
+Making install in m4
+make[1]: Entering directory '/tmp/cracklib-2.9.7/m4'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/m4'
+make[2]: Nothing to be done for 'install-exec-am'.
+make[2]: Nothing to be done for 'install-data-am'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/m4'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/m4'
+Making install in lib
+make[1]: Entering directory '/tmp/cracklib-2.9.7/lib'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/lib'
+ /usr/bin/mkdir -p '/tmp/package-cracklib/usr/lib'
+ /bin/sh ../libtool   --mode=install /usr/bin/ginstall -c   libcrack.la '/tmp/package-cracklib/usr/lib'
+libtool: install: /usr/bin/ginstall -c .libs/libcrack.so.2.9.0 /tmp/package-cracklib/usr/lib/libcrack.so.2.9.0
+libtool: install: (cd /tmp/package-cracklib/usr/lib && { ln -s -f libcrack.so.2.9.0 libcrack.so.2 || { rm -f libcrack.so.2 && ln -s libcrack.so.2.9.0 libcrack.so.2; }; })
+libtool: install: (cd /tmp/package-cracklib/usr/lib && { ln -s -f libcrack.so.2.9.0 libcrack.so || { rm -f libcrack.so && ln -s libcrack.so.2.9.0 libcrack.so; }; })
+libtool: install: /usr/bin/ginstall -c .libs/libcrack.lai /tmp/package-cracklib/usr/lib/libcrack.la
+libtool: warning: remember to run 'libtool --finish /usr/lib'
+ /usr/bin/mkdir -p '/tmp/package-cracklib/usr/include'
+ /usr/bin/ginstall -c -m 644 crack.h packer.h '/tmp/package-cracklib/usr/include'
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/lib'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/lib'
+Making install in util
+make[1]: Entering directory '/tmp/cracklib-2.9.7/util'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/util'
+ /usr/bin/mkdir -p '/tmp/package-cracklib/sbin'
+ /usr/bin/ginstall -c create-cracklib-dict cracklib-format '/tmp/package-cracklib/sbin'
+ /usr/bin/mkdir -p '/tmp/package-cracklib/sbin'
+  /bin/sh ../libtool   --mode=install /usr/bin/ginstall -c cracklib-packer cracklib-unpacker cracklib-check '/tmp/package-cracklib/sbin'
+libtool: warning: '../lib/libcrack.la' has not been installed in '/usr/lib'
+libtool: install: /usr/bin/ginstall -c .libs/cracklib-packer /tmp/package-cracklib/sbin/cracklib-packer
+libtool: warning: '../lib/libcrack.la' has not been installed in '/usr/lib'
+libtool: install: /usr/bin/ginstall -c .libs/cracklib-unpacker /tmp/package-cracklib/sbin/cracklib-unpacker
+libtool: warning: '../lib/libcrack.la' has not been installed in '/usr/lib'
+libtool: install: /usr/bin/ginstall -c .libs/cracklib-check /tmp/package-cracklib/sbin/cracklib-check
+make[2]: Nothing to be done for 'install-data-am'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/util'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/util'
+Making install in po
+make[1]: Entering directory '/tmp/cracklib-2.9.7/po'
+if test "cracklib" = "gettext-tools"; then \
+  /usr/bin/mkdir -p /tmp/package-cracklib/usr/share/gettext/po; \
+  for file in Makefile.in.in remove-potcdate.sin quot.sed boldquot.sed en@quot.header en@boldquot.header insert-header.sin Rules-quot   Makevars.template; do \
+    /usr/bin/ginstall -c -m 644 ./$file \
+		    /tmp/package-cracklib/usr/share/gettext/po/$file; \
+  done; \
+  for file in Makevars; do \
+    rm -f /tmp/package-cracklib/usr/share/gettext/po/$file; \
+  done; \
+else \
+  : ; \
+fi
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/po'
+Making install in doc
+make[1]: Entering directory '/tmp/cracklib-2.9.7/doc'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/doc'
+make[2]: Nothing to be done for 'install-exec-am'.
+ /usr/bin/mkdir -p '/tmp/package-cracklib/usr/share/cracklib'
+ /usr/bin/ginstall -c -m 644 cracklib.magic '/tmp/package-cracklib/usr/share/cracklib'
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/doc'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/doc'
+Making install in python
+make[1]: Entering directory '/tmp/cracklib-2.9.7/python'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/python'
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/python'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/python'
+Making install in dicts
+make[1]: Entering directory '/tmp/cracklib-2.9.7/dicts'
+make[2]: Entering directory '/tmp/cracklib-2.9.7/dicts'
+make[2]: Nothing to be done for 'install-exec-am'.
+ /usr/bin/mkdir -p '/tmp/package-cracklib/usr/share/cracklib'
+ /usr/bin/ginstall -c -m 644 cracklib-small '/tmp/package-cracklib/usr/share/cracklib'
+make[2]: Leaving directory '/tmp/cracklib-2.9.7/dicts'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7/dicts'
+make[1]: Entering directory '/tmp/cracklib-2.9.7'
+
+=======================================================
+Be sure and obtain a large wordlist file and run
+make install and make dict to generate the word
+list index file. See the README file for more details.
+=======================================================
+
+make[2]: Entering directory '/tmp/cracklib-2.9.7'
+make[2]: Nothing to be done for 'install-exec-am'.
+make[2]: Nothing to be done for 'install-data-am'.
+make[2]: Leaving directory '/tmp/cracklib-2.9.7'
+make[1]: Leaving directory '/tmp/cracklib-2.9.7'
+51526 51526
+
+Slackware package maker, version 3.14159265.
+
+Searching for symbolic links:
+usr/lib/libcrack.so	libcrack.so.2.9.0
+usr/lib/libcrack.so.2	libcrack.so.2.9.0
+var/cache/cracklib/cracklib_dict.pwd	/usr/share/cracklib/pw_dict.pwd
+var/cache/cracklib/cracklib_dict.pwi	/usr/share/cracklib/pw_dict.pwi
+
+Making symbolic link creation script:
+( cd usr/lib ; rm -rf libcrack.so )
+( cd usr/lib ; ln -sf libcrack.so.2.9.0 libcrack.so )
+( cd usr/lib ; rm -rf libcrack.so.2 )
+( cd usr/lib ; ln -sf libcrack.so.2.9.0 libcrack.so.2 )
+( cd var/cache/cracklib ; rm -rf cracklib_dict.pwd )
+( cd var/cache/cracklib ; ln -sf /usr/share/cracklib/pw_dict.pwd cracklib_dict.pwd )
+( cd var/cache/cracklib ; rm -rf cracklib_dict.pwi )
+( cd var/cache/cracklib ; ln -sf /usr/share/cracklib/pw_dict.pwi cracklib_dict.pwi )
+
+It is recommended that you make these lines your new installation script.
+
+Would you like to make this stuff the install script for this package
+and remove the symbolic links ([y]es, [n]o)? y
+
+
+Removing symbolic links:
+removed './usr/lib/libcrack.so.2'
+removed './usr/lib/libcrack.so'
+removed './var/cache/cracklib/cracklib_dict.pwi'
+removed './var/cache/cracklib/cracklib_dict.pwd'
+
+Creating your new ./install/doinst.sh...
+
+This next step is optional - you can set the directories in your package
+to some sane permissions. If any of the directories in your package have
+special permissions, then DO NOT reset them here!
+
+Would you like to reset all directory permissions to 755 (drwxr-xr-x) and
+directory ownerships to root.root ([y]es, [n]o)? n
+
+Creating Slackware package:  /tmp/cracklib-2.9.7-x86_64-1.txz
+
+./
+install/
+install/doinst.sh
+install/slack-desc
+sbin/
+sbin/cracklib-check
+sbin/cracklib-format
+sbin/cracklib-packer
+sbin/cracklib-unpacker
+sbin/create-cracklib-dict
+usr/
+usr/doc/
+usr/doc/cracklib-2.9.7/
+usr/doc/cracklib-2.9.7/AUTHORS
+usr/doc/cracklib-2.9.7/COPYING.LIB
+usr/doc/cracklib-2.9.7/ChangeLog
+usr/doc/cracklib-2.9.7/INSTALL
+usr/doc/cracklib-2.9.7/NEWS
+usr/doc/cracklib-2.9.7/README
+usr/doc/cracklib-2.9.7/README-DAWG
+usr/doc/cracklib-2.9.7/README-LICENSE
+usr/doc/cracklib-2.9.7/README-WORDS
+usr/include/
+usr/include/crack.h
+usr/include/packer.h
+usr/lib/
+usr/lib/libcrack.so.2.9.0
+usr/share/
+usr/share/cracklib/
+usr/share/cracklib/cracklib-small
+usr/share/cracklib/cracklib.magic
+usr/share/cracklib/pw_dict.hwm
+usr/share/cracklib/pw_dict.pwd
+usr/share/cracklib/pw_dict.pwi
+var/
+var/cache/
+var/cache/cracklib/
+WARNING:  zero length file usr/doc/cracklib-2.9.7/ChangeLog
+
+Slackware package /tmp/cracklib-2.9.7-x86_64-1.txz created.
+
